packages/time-picker/src/TimePicker/TimePicker.spec.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from "react"
+import { fireEvent, render, screen } from "@testing-library/react"
+import { TimePicker, TimePickerProps } from "./TimePicker"
+
+const mockSetValue = jest.fn()
+const UTC_ZERO_TIMEZONE = "Africa/Abidjan"
+
+const pressArrowKey =
+  (direction: "ArrowUp" | "ArrowDown") => (element: HTMLElement) =>
```

Comment:
It doesn’t really matter but I’m curious why this is a curried function? Why not have two arguments?

Reply:
I thought i was going to be clever and have a bunch of different functions for different elements and keystrokes. But didn't really work out that way.